File: src/test/java/com/github/natche/gravatarjavaclient/TestingImageUrls.java
package com.github.natche.gravatarjavaclient;

/**
 * Constants related to testing.
 */
public final class TestingImageUrls {
    public static final String foreignImageUrl = "https://picsum.photos/seed/gravatar-java-client/200/300";
    public static final String anotherForeignImageUrl = "https://picsum.photos/seed/gravatar-java-client/300/300";

    /**
     * Suppress default constructor.
     */
    private TestingImageUrls() {
        throw new AssertionError("Cannot create instances of TestingConstants");
    }
}

File: src/test/java/com/github/natche/gravatarjavaclient/TestingImageUrls.kt
package com.github.natche.gravatarjavaclient

/**
 * Constants related to testing.
 */
class TestingImageUrls private constructor() {
    /**
     * Suppress default constructor.
     */
    init {
        throw AssertionError("Cannot create instances of TestingConstants")
    }

    companion object {
        const val foreignImageUrl = "https://picsum.photos/seed/gravatar-java-client/200/300"
        const val anotherForeignImageUrl = "https://picsum.photos/seed/gravatar-java-client/300/300"
    }
}